src/target/riscv/riscv-013.c

```diff
@@ -285,6 +286,31 @@ static dm013_info_t *get_dm(struct target *target)
 	return dm;
 }
 
+static riscv_debug_reg_ctx_t get_riscv_debug_reg_ctx(struct target *target)
+{
+	RISCV_INFO(r);
+	RISCV013_INFO(info);
+	const riscv_debug_reg_ctx_t context = {
+		.XLEN = { .value = r->xlen, .is_set = true },
+		.DXLEN = { .value = r->xlen, .is_set = true },
+		.abits = { .value = info->abits, .is_set = true },
+	};
+	return context;
+}
+
+#define RISCV_LOG_REG(target, reg, value) \
+do {\
```

Comment:
Can this be a function? I don't think the performance is so critical, and it makes the code easier to deal with.

Reply:
Addressed.